features/other_b/bookmark_indexing.feature

```diff
@@ -60,48 +60,59 @@ Feature: Bookmark Indexing
     Then I should not see "Series"
 
   @new-search
-  Scenario: Synning a canonical tag used on a bookmarked series should move the 
-  bookmark to the new canonical's bookmark listings; de-synning should remove it
+  Scenario: Synning a canonical tag used on bookmarked series and external works
+  should move the bookmarks to the new canonical's bookmark listings; de-synning
+  should remove them
     Given I am logged in as "author"
       And a canonical fandom "Veronica Mars"
```

Comment:
Is see these step names are a bit light on readability - this reads `Given I am logged in as "author" and a canonical character "Veronica Mars"` - probably lacking an "exists" in the step definition. But that's a pre-existing problem and it isn't as if we have a "business" that actually reads these scenarios!